### PR TITLE
Learn modular images updates

### PR DESCRIPTION
**Pull request by alex-beckett**
Update images for:
- Basics of modular blockchains
- Modular architectures

Also swapped the conclusion into a summary for:
- Basics of modular blockchains
- Modular architectures
- Modular scalability
- creating new blockchains

Other misc edits
- adding links
- fixed sentences

### CODE REVIEW

File: src/pages/markdown-pages/learn/modular-architectures.md [hunk 1]
```diff
@@ -17,65 +17,56 @@ description: "A look into the different ways in which modular blockchains can be
...
+2. Optimized modular stacks split the functions across multiple specialized blockchains. These modular stacks can take multiple forms to serve varying goals.
+3. A modular stack can also use three layers, where an execution layer sits atop a settlement layer with an underlying data availability layer. This stack can introduce other desireable features, like bridging and liquidity.
 
 #### Introduction
 
-A new paradigm is emerging in which modular blockchains are enabling new chains to be constructed in ways that were not previously possible. Because of this, the design possibilities are vast for both the individual blockchain and the modular stack it is a part of. Different types of modular blockchains can work synergistically, varying by purpose and architecture.
+A new paradigm is emerging in which modular blockchains are enabling new chains to be constructed in ways that were not previously possible. Because of this, the design possibilities are vast for both the individual blockchain and the modular stack it is a part of. Different types of modular blockchains can work collaborative manner, varying by purpose and architecture.
```
Comment (dabit3) [inline src/pages/markdown-pages/learn/modular-architectures.md]: maybe "can work in a collaborative manner"

File: src/pages/markdown-pages/learn/modular-scalability.md [hunk 2]
```diff
@@ -17,37 +17,39 @@ description: "The scalability properties of different types of modular blockchai
   <meta name="twitter:image" content="https://raw.githubusercontent.com/celestiaorg/celestia.org/main/src/pages/markdown-pages/learn/images/learn-modular-twitter-card.png">
 <head/>
 
+#### Summary
+1. The two main designs of rollups are optimistic and zk-Rollups. Optimistic rollups provide scalability through data compression and alleviating state growth on the base layer. zk-Rollups scale similarly, as well as through efficient compute verification with validity proofs.
+2. Data availability layers scale block verification and minimize bandwidth costs for light nodes. This is possible with data availability sampling, which allows light nodes to verify data availability by only downloading small random samples of a block.
+3. Settlement layers can scale using the same techniques from execution and data availability layers. In their current form, settlement layers are unideal. A more optimal approach would be to create an environment that limits smart contracts to favor interactions with execution layers.
```
Comment (dabit3) [inline src/pages/markdown-pages/learn/modular-scalability.md]: Is this essentially saying a settlement layer optimized for rollups / batch transactions? if so there might be some opportunity to be more clear about this.

File: src/pages/markdown-pages/learn/modular-scalability.md [hunk 3]
```diff
@@ -68,15 +70,4 @@ The current scope of settlement layers has been borne out of necessity for a blo
...
-</div>
+The vast majority of the settlement layers transaction activity should consist of execution layers settling transactions. <a href="https://forum.celestia.org/t/increasing-scalability-of-the-evm-for-rollups-by-restricting-state-and-contract-set/78" target="_blank" rel="noopener noreferrer" style="color:#7B2BF9;">One method</a> of influencing this behavior is to limit smart contracts such that they are only used to interact with execution layers, providing proof verification and bridging tokens. By doing so an environment can be created that is conducive to scaling a settlement layer. Many techniques used to scale execution and data availability layers are also applicable to the settlement layer, including mechanics such as parallel transaction processing, fraud or validity proof, <a href="https://forum.celestia.org/t/the-state-growth-problem-in-a-modular-blockchain-ecosystem/325" target="_blank" rel="noopener noreferrer" style="color:#7B2BF9;">weak statelessness</a>, and <a href="https://forum.celestia.org/t/the-state-growth-problem-in-a-modular-blockchain-ecosystem/325" target="_blank" rel="noopener noreferrer" style="color:#7B2BF9;">history expiry</a>.
```
Comment (dabit3) [inline src/pages/markdown-pages/learn/modular-scalability.md:73]: This is such a great paragraph.

**Review by dabit3 — APPROVED** [review]
Looks great, left a few minor comments.

**Comment by alex-beckett** [comment]
thx for the review ser @dabit3

**Pull request merged**